scripts/docker/ubuntu.14.04/Dockerfile

```diff
@@ -7,64 +7,40 @@
 FROM ubuntu:14.04
 
 # Misc Dependencies for build
-RUN apt-get update && apt-get -qqy install curl unzip gettext sudo
+RUN apt-get update && \
+    apt-get -qqy install \
+        curl \
+        unzip \
+        gettext \
+        sudo && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
 
 # This could become a "microsoft/coreclr" image, since it just installs the dependencies for CoreCLR (and stdlib)
-RUN echo "deb http://llvm.org/apt/trusty/ llvm-toolchain-trusty-3.6 main"  | tee /etc/apt/sources.list.d/llvm.list && \
-    curl http://llvm.org/apt/llvm-snapshot.gpg.key | apt-key add - && \
-    apt-get update && apt-get -qqy install\
-	libc6 \
-	libedit2 \
-	libffi6 \
-	libgcc1 \
-	libicu52 \
-	liblldb-3.6 \
-	libllvm3.6 \
-	liblttng-ust0 \
-	liblzma5 \
-	libncurses5 \
-	libpython2.7 \
-	libstdc++6 \
-	libtinfo5 \
-	libunwind8 \
-	liburcu1 \
-	libuuid1 \
-	zlib1g \
-	libasn1-8-heimdal \
-	libcomerr2 \
-	libcurl3 \
-	libgcrypt11 \
-	libgnutls26 \
-	libgpg-error0 \
-	libgssapi3-heimdal \
-	libgssapi-krb5-2 \
-	libhcrypto4-heimdal \
-	libheimbase1-heimdal \
-	libheimntlm0-heimdal \
-	libhx509-5-heimdal \
-	libidn11 \
-	libk5crypto3 \
-	libkeyutils1 \
-	libkrb5-26-heimdal \
-	libkrb5-3 \
-	libkrb5support0 \
-	libldap-2.4-2 \
-	libp11-kit0 \
-	libroken18-heimdal \
-	librtmp0 \
-	libsasl2-2 \
-	libsqlite3-0 \
-	libssl1.0.0 \
-       libssl-dev \
-	libtasn1-6 \
-	libwind0-heimdal
-
-# Install Dotnet CLI dependencies.
-# clang is required for dotnet-compile-native
-RUN apt-get -qqy install clang-3.5
+RUN apt-get update && \
```

Comment:
Could you also clean up the dependencies here: 
https://github.com/dotnet/core-setup/blob/master/build_projects/shared-build-targets-utils/PackageDependencies.cs#L24